### PR TITLE
Users/mithunj/ampmissue

### PR DESCRIPTION
**Pull request by mithunjohny**
### Description
_Include a description of the problem to be solved_

## Solution Proposed
Some reason browser is not able to properly set direction to bot timestamp , so adding a span around timestamp  solve the issue

### Acceptance criteria
_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_
When direction is rtl
image
when direction is default
image
device locale
image

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [x] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

**Review by xTEddie — APPROVED** [review]

File: CHANGE_LOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Fixed
-
+- Added addtional span around time part of bubble timestamp
```
Comment (xTEddie) [inline CHANGE_LOG.md:10]: `additional`

**Review by elopezanaya — APPROVED** [review]

File: chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/timestamps/DeliveredTimestamp.tsx [hunk 2]
```diff
@@ -33,6 +33,8 @@ export const DeliveredTimestamp = ({ args, role, name }: any) => {
         // make sure the "rtl" direction doesn't produce "PM 1:23", but remains "1:23 PM"
         if (dir === "rtl" && isAmPmFormat) {
             return <span dir="ltr">{getTimestampHourMinute(timestamp)}</span>;
+        } else {
+            return <span dir={dir}>{getTimestampHourMinute(timestamp)}</span>;
```
Comment (dleblondMSFT) [inline chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/timestamps/DeliveredTimestamp.tsx:37]: do we need the IF or can we use this all the time?

Comment (mithunjohny) [reply chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/timestamps/DeliveredTimestamp.tsx:37]: Yes we need to handle scenario when rtl is enabled but OS is on english. "PM 1:23", but remains "1:23 PM" ,refer first screen shot
